bundles/org.openhab.transform.bin2json/NOTICE

```diff
@@ -0,0 +1,20 @@
+This content is produced and maintained by the openHAB project.
+
+* Project home: https://www.openhab.org
+
+== Declared Project Licenses
+
+This program and the accompanying materials are made available under the terms
+of the Eclipse Public License 2.0 which is available at
+https://www.eclipse.org/legal/epl-2.0/.
+
+== Source Code
+
+https://github.com/openhab/openhab2-addons
+
+== Third-party Content
+
+jbbp
```

Comment:
Can the `jbbp-1.4.1.jar` library be provided by maven as dependency instead of adding the jar file?